### PR TITLE
snapper: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
- `snapper 0.10.3`

### CODE REVIEW

**Comment by isaacvicente** [comment]
As suggested by @kbdharun, I translated the main English page to a Portuguese one, so the consistency is maintained.

**Review by kbdharun — COMMENTED** [review]
The translation LGTM, Thanks for your contribution ❤️.

I have a doubt about this word, `configuração` or `configuraç` which one is used for `config`?  Can one term be used here throughout the page?

**Comment by isaacvicente** [comment]
It's "configuração". I think I missed some keys xD. Can you modify "configuraç" to "configuração", please?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.pt_BR/linux/snapper.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# snapper
+
+> Ferramenta de gerenciamento de snapshots do sistema de arquivos.
+> Mais informações: <http://snapper.io/manpages/snapper.html>.
+
+- Listar configurações de snapshots:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/snapper.md:6]:
```suggestion
- Lista configurações de snapshots:
```

File: pages.pt_BR/linux/snapper.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`snapper list-configs`
+
+- Criar configuração do snapper:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/snapper.md:10]:
```suggestion
- Cria configuração do snapper:
```

File: pages.pt_BR/linux/snapper.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`snapper -c {{configuração}} create-config {{caminho/para/diretório}}`
+
+- Criar um snapshot com uma descrição:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/snapper.md:14]:
```suggestion
- Cria um snapshot com uma descrição:
```

File: pages.pt_BR/linux/snapper.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`snapper -c {{configuração}} create -d "{{descrição_do_snapshot}}"`
+
+- Listar snapshots para uma configuração:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/snapper.md:18]:
```suggestion
- Lista snapshots para uma configuração:
```

File: pages.pt_BR/linux/snapper.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`snapper -c {{configuração}} list`
+
+- Excluir um snapshot:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/snapper.md:22]:
```suggestion
- Exclue um snapshot:
```

File: pages.pt_BR/linux/snapper.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`snapper -c {{configuração}} delete {{número_do_snapshot}}`
+
+- Excluir um intervalo de snapshots:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/snapper.md:26]:
```suggestion
- Exclue um intervalo de snapshots:
```

**Review by navarroaxel — APPROVED** [review]

**Comment by isaacvicente** [comment]
This PR is “not accepted” somehow in my Hacktoberfest profile. Any reason this occurs?

**Comment by kbdharun** [comment]
> This PR is “not accepted” somehow in my Hacktoberfest profile. Any reason this occurs?

It takes 15 minutes for your hacktoberfest profile to refresh if your visit the page often. But it usually takes about 6 hours in most cases. This PR will be reflected there soon.

**Comment by isaacvicente** [comment]
> > This PR is “not accepted” somehow in my Hacktoberfest profile. Any reason this occurs?
> 
> It takes 15 minutes for your hacktoberfest profile to refresh if your visit the page often. But it usually takes about 6 hours in most cases. This PR will be reflected there soon.

Yeah, some of my PR has a "waiting" status and a timer. I will wait these 15 min then, so the "waiting" thing appears.

**Comment by kbdharun** [comment]
> > > This PR is “not accepted” somehow in my Hacktoberfest profile. Any reason this occurs?
> > 
> > 
> > It takes 15 minutes for your hacktoberfest profile to refresh if your visit the page often. But it usually takes about 6 hours in most cases. This PR will be reflected there soon.
> 
> Yeah, some of my PR has a "waiting" status and a timer. I will wait these 15 min then, so the "waiting" thing appears.

Actually, Even if the PR is merged here, the team from Hackroberfest would review your contribution I guess within 7 days. If timer is shown then your PR is merged and is ready for their review.

**Pull request merged**